### DISCUSSION TITLE
Where can i learn more?

### DISCUSSION THREAD

**Discussion by EnzoDiazDev** [uNetworking/uWebSockets.js#731] [Q&A] [answered]
Where can I learn more about the different compression options? What are the pros vs cons of the different options?
Where can I learn more about Backpressure, and what would be the best way to deal with it?
I don't know exactly how to start looking into this, any clues or documentation would be helpful.

Thank you!

**Comment by e3dio** [comment, marked as answer]
https://github.com/uNetworking/uWebSockets/blob/master/misc/READMORE.md

**Reply by EnzoDiazDev** [reply]
Oh sorry, that was more obvious than I expected.
Thank you so much!
